scriptmodules/libretrocores/lr-fceumm.sh

```diff
@@ -41,8 +41,11 @@ function configure_lr-fceumm() {
     rm -rf "$rootdir/$md_type/lr-fceu-next"
 
     mkRomDir "nes"
+    mkRomDir "fds"
     ensureSystemretroconfig "nes"
+    ensureSystemretroconfig "fds"
 
     delSystem "lr-fceu-next" "nes"
-    addSystem 1 "$md_id" "nes" "$md_inst/fceumm_libretro.so"
+    addSystem 0 "$md_id" "nes" "$md_inst/fceumm_libretro.so"
+    addSystem 1 "$md_id" "fds" "$md_inst/fceumm_libretro.so"
 }
```

Comment:
addSystem 0 means it's not set as default - addSystem 1 means it is (Although existing defaults are never changed)

So in the case above you have made it so fceumm is no longer the default nes emulator which is wrong. It could be the default fds emulator if it is tested to be better than the other.